lib/libs/api/package/itemExists.ts

```diff
@@ -5,15 +5,10 @@ export async function itemExists(params: {
   osDomain?: string;
   indexNamespace?: string;
 }): Promise<boolean> {
-  try {
```

Comment:
removed try/catch so it can be handled by the calling function and the errors can be treated appropriately